### ISSUE TITLE
load and save pre-trained model

### ISSUE THREAD

**Issue by trentwall** [tensorflow/cleverhans#1160]
I would like to load a pre-trained model, so that I can get the same accuracy and success rate of adversarial example generation I achieved before saving the model, with out having to retrain the loaded model.

The current save function seems to only save the initial values for the weights and biases not the current values. After loading a model I have to retrain it to achieve the expected success rates.
